test/sim/abilities/sheerforce.js

```diff
@@ -43,4 +43,20 @@ describe('Sheer Force', function () {
 		battle.makeChoices('move bodyslam', 'move rest');
 		assert.strictEqual(battle.p1.active[0].hp, battle.p1.active[0].maxhp);
 	});
+
+	it('should not eliminate the effect of a Max Move', function () {
+		battle = common.createBattle();
+		battle.setPlayer('p1', {team: [
+			{species: 'Braviary', ability: 'sheerforce', moves: ['heatwave', 'facade', 'superpower']},
+		]});
+		battle.setPlayer('p2', {team: [
+			{species: 'Shedinja', ability: 'sturdy', item: 'ringtarget', moves: ['splash']},
+		]});
+		battle.makeChoices('move heatwave dynamax', 'auto');
```

Comment:
Travis build errors here: error, "Error: [Invalid choice] Can't move: You can only Dynamax once per battle."

Reply:
It doesn't error here anymore; I presume some recent update fixed this.